dns/octodns-docker/Dockerfile

```diff
@@ -12,9 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# python:2-alpine as of July 11th, 2019
-ARG BASE_IMAGE=python@sha256:9a2db929c9e4c65118cb1da9ddb1d64f3ba8e4a722f07f3f2d34b58b1e8556b1
-FROM ${BASE_IMAGE}
+# python:3.7.5-alpine3.10 as of November 14th, 2019
```

Comment:
What's the reason for selecting this tag, as opposed to `python:3-alpine`?

Reply:
I tried to mimic the way the last image was selected, the latest image and then frozen with the digest.
It seems that no changes are needed to check-zone.py but io can only test it so far

Reply:
Yes, but the way the tag the digest was based off of was `python:2-alpine`, as noted in the comment. Could we not use `python:3-alpine` to select the new digest, instead of `python:3.7.5-alpine3.10`?

Reply:
OK, I've added a new commit with the new image digest for 3-alpine which currently corresponds to 3.8.0-alpine3.10.